config/routes.rb

```diff
@@ -12,8 +12,12 @@
     get "/accessibility-statement", to: "accessibility_statement#show"
 
     # Redirect for deleted question and page (301 is default)
-    get "/urgent-medical-help", to: redirect("/need-help-with")
-    get "/get-help-from-nhs", to: redirect("/need-help-with")
+    get "/urgent-medical-help", to: redirect("/where-live")
+    get "/get-help-from-nhs", to: redirect("/where-live")
+
+    # Question: Where do you live
+    get "/where-live", to: "where_live#show"
```

Comment:
I wonder if the route should just be a single word (like `location`) rather than part of a sentence?

Reply:
So i've called the group that this question is in location just in case we need to add any more granularity. How would you feel about calling this "country"?

Reply:
just seen later comments, how do you feel about "nation"? :D

Reply:
Just seen later comments, how do you feel about "nation"? :D

Reply:
Nation is good.  I just wanted to avoid any potential controversy over the country status of the four constituents of the UK.